src/app/lists/ListsPage.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
```

Comment:
The logic in this file was moved out of lists/page.tsx and adjusted so that it stops hydration errors